modules/spo-replicating-controllers.adoc

```diff
@@ -112,15 +100,4 @@ spec:
 The SELinux type is not specified in the workload and is handled by the SCC. When the pods are created by the deployment and the `ReplicaSet`, the pods will run with the appropriate profile.
 ====
 
-Ensure your SCC is only usable by the correct service account. Refer to _Additional resources_ for more information.
-
-ifeval::["{context}" == "spo-seccomp"]
-:!seccomp:
-:!type:
-:!kind:
-endif::[]
-ifeval::["{context}" == "spo-selinux"]
-:!selinux:
-:!type:
-:!kind:
-endif::[]
\ No newline at end of file
+Ensure your SCC is only usable by the correct service account. Refer to _Additional resources_ for more information.
```

Comment:
```suggestion
Ensure that your SCC is usable by only the correct service account. Refer to _Additional resources_ for more information.
```
Place "only" as close to the word it modifies as possible: https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only